include/deal.II/fe/fe_nedelec.h

```diff
@@ -139,6 +139,253 @@ DEAL_II_NAMESPACE_OPEN
  * The points needed are those of QGauss<sub>k+1</sub> on each edge and
  * QGauss<sub>k+2</sub> on each face and in the interior of the cell (or none
  * for N<sub>1</sub>).
+ *
+ * <h4>FE_Nedelec(0)</h4>
```

Comment:
```suggestion
 * <h3> Depictions of shape functions <h3>
 * The following subsections show depictions of the shape functions defined by this class. The
 * main piece of information in the figures below are the vectors that illustrate direction and magnitude
 * of shape functions. These vectors are overlaid to a surface and color plot where both surface elevation
 * and the intensity of the color indicate the *magnitude* of the vector-valued shape function. Note that
 * the surface plot does not always reflect the smoothness of shape functions: Even if $\vec\varphi_i(\vec x)$
 * is a smooth function, $\|\vec\varphi_i(\vec x)\|$ will in general not be at places where $\vec\varphi_i(\vec x)$
 * is zero.
 *
 * <h4>FE_Nedelec(0)</h4>
```

Reply:
Yes, the issue of the "artificial" non-smoothness in the illustrations was something I debated for awhile to decide which representation was better. Based on your comment below, I switched to a 2D representation. Please let me know which realization you prefer and then I can squash the commits.

I could also elevate based on the sum of the components (rather than the magnitude), since only one component is non-zero for any given shape function (for the unit cell; this is not true in general).

Reply:
I like the 2d plots much better. Let's go with that. Can you still add the h3 header (and, if you'd like some of the text) to the documentation?

Reply:
Great! I included the h3 header and what I thought was relevant given that I am no longer including the magnitude/elevation. (I've never used "commit suggestion" via github before, so I just added the change in manually in my previous commit)

Reply:
Doing it by hand is certainly reasonable. I think you might still have to push these changes. You might also want to squash the patches so that we don't have both versions of the images in the repository.